src/components/notifications/HdNotificationsBar.vue

```diff
@@ -1,19 +1,18 @@
 <template>
   <div
-    :class="{
-      'notifications-bar': true,
-      [`notifications-bar--${type}`]: true,
-      'notifications-bar--visible': visible,
-      'notifications-bar--compact': compact,
-    }"
+    class="notifications-bar"
+    :class="[`notifications-bar--${type}`]"
   >
-    <div class="notifications-bar__icon">
-      <HdIcon
-        :src="icon"
-        class="notifications-bar__icon__image"
-      />
-    </div>
-    <div class="notifications-bar__content">
+    <HdIcon
+      :src="icon"
+      class="notifications-bar__icon"
+    />
+    <div
+      class="notifications-bar__content"
+      :class="{
+        'notifications-bar__content--centered': centered
```

Comment:
I've used `centered` because the component content only has two states:
- Default
- Centered

The default is left aligned and the dev can choose to center align.
So you can then set it by just passing the prop:

```
<HdNotificationTab centered />

// instead of

<HdNotificationTab alignment="center" />
```

You can find more details by looking in the documentation:
https://homeday.zeroheight.com/styleguide/s/22414/p/79cf20-notification

But I can change the prop to be called alignment, that's not a problem =D

Reply:
So, centered is enough. 👍